examples/metrics_datadog/sam/template.yaml

```diff
@@ -0,0 +1,31 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Transform: AWS::Serverless-2016-10-31
+Description: Powertools for AWS Lambda (Python) version
+
+Globals:
+  Function:
+    Timeout: 5
+    Runtime: python3.10
+    Tracing: Active
+    Environment:
+      Variables:
+        POWERTOOLS_METRICS_NAMESPACE: ServerlessAirline
+        DD_API_KEY: "<YOUR DATADOG API KEY>"
+        DD_SITE: datadoghq.com
```

Comment:
add a quick comment on the right OR even better... a code annotation explaining what this is along with a link do Datadog docs.